ui/perfherder/Validation.jsx

```diff
@@ -41,6 +41,15 @@ const withValidation = (
       this.validateParams(parseQueryParams(this.props.location.search));
     }
 
+    shouldComponentUpdate(prevProps) {
+      const { location } = this.props;
+
+      if (location.hash !== prevProps.location.hash) {
```

Comment:
Are you able to just return the result of the check? :

    return location.hash === prevProps.location.hash;

Reply:
Good point - that should work.